0.download_data/download_figshare.py

```diff
@@ -45,5 +45,8 @@ def download_figshare(
     # move the metadata files into a new folder from the images
     for files in output_dir.iterdir():
```

Comment:
when using a for loop, it is typical for the iterator to be singular (`file`) instead of plural (`files`). You may want to improve the variable name. `files` is not very informative! What type of files?

Also, it looks like you're looping through every single image 😨 It's probably ok for NF1 since there are not many images, but can you think of a way to _not_ need to loop?

Reply:
That is a great point since plate 3 will have a LOT more images. What I decided on doing after some testing is using glob; see commit [4d4f79f](https://github.com/WayScience/nf1_cellpainting_data/pull/2/commits/4d4f79ff6eb2f5c52c0a543db3968b3c6fbf437f) for the change I made.